composer.json

```diff
@@ -13,12 +13,26 @@
             "role": "Developer"
         }
     ],
+    "repositories":[
```

Comment:
Is this part necessary?

Reply:
Currently, yes. The `coding-standards` is not published to packagist.

Reply:
The packagist package exists now and this repository specification has been removed.